README.md

```diff
@@ -17,6 +17,24 @@
 
 `pip install git+https://github.com/Tomme/dbt-athena.git`
 
+### Prerequisites
+
+To start, you will need an S3 bucket, for instance `my-staging-bucket` and an Athena database:
```

Comment:
You can also create databases via AWS Glue: https://aws.amazon.com/glue/?whats-new-cards.sort-by=item.additionalFields.postDateTime&whats-new-cards.sort-order=desc Might be worth adding a footnote about that as well.

Reply:
Added! 😄 ✅